src/common/arrstr.cpp

```diff
@@ -27,6 +27,17 @@
 #include <functional>
 #include "wx/afterstd.h"
 
+#include "wx/wxcrt.h"
+
+#if wxUSE_REGEX
```

Comment:
These guards are not needed, they are already inside `wx/regex.h`

Reply:
Sigh. I even looked at other sources how they do it but looked only at src/msw/textctrl.cpp which has
```cpp
#if wxUSE_CLIPBOARD
    #include "wx/clipbrd.h"
    #include "wx/dataobj.h"
#endif
```